dev-tools/mage/files/packages.yml

```diff
@@ -0,0 +1,222 @@
+---
+
+# This file contains the package specifications for both Community Beats and
+# Official Beats. The shared section contains YAML anchors that are used to
+# define common parts of the package in order to not repeat ourselves.
+
+shared:
+- &common
+  name:         '{{.BeatName}}'
+  service_name: '{{.BeatServiceName}}'
+  os:           '{{.GOOS}}'
+  arch:         '{{.PackageArch}}'
+  vendor:       '{{.BeatVendor}}'
+  version:      '{{ beat_version }}'
+  license:      '{{.BeatLicense}}'
+  url:          '{{.BeatURL}}'
+  description:  '{{.BeatDescription}}'
```

Comment:
For community Beats they will have to move the description to the Golang file?

Reply:
Yes. Or `export BEAT_DESCRIPTION` from their Makefile. But I'd prefer the `magefile.go` to be the source of truth for this information (or an external YAML file as you suggested) so that `make` isn't a requirement for building.